Dockerfile

```diff
@@ -28,47 +23,45 @@ RUN apt-get install -y -q eatmydata
 # Some rudimentary tools if we need to do anything within docker and curl and unzip needed for setting up conda
 RUN bash -c "eatmydata apt-get install -y -q vim less man-db curl unzip bzip2"
 # Run additional lines, primarily to setup/enable snapshots repository etc
-RUN bash -c "sed -i -e 's,http://neuro.debian.net/debian/* ,http://snapshot-neuro.debian.net:5002/archive/neurodebian/20150306T060524Z/ ,g' etc/apt/sources.list.d/neurodebian.sources.list;"
+RUN bash -c "sed -i -e 's,http://neuro.debian.net/debian/* ,http://snapshot-neuro.debian.net:5002/archive/neurodebian/20170410T000000Z/ ,g' etc/apt/sources.list.d/neurodebian.sources.list"
 RUN bash -c "curl -s http://neuro.debian.net/_files/knock-snapshots;"
 RUN bash -c "eatmydata apt-get update; eatmydata apt-get dist-upgrade -y;"
-# Install fsl-complete
-#RUN bash -c "eatmydata apt-get install -y -q fsl-complete"
 # We might be just fine with the core here
 RUN bash -c "eatmydata apt-get install -y -q fsl-core fsl-first-data"
 
-RUN apt-get clean
+RUN apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /boot /media /mnt /srv
+
+ENV FSLMULTIFILEQUIT=TRUE
+ENV POSSUMDIR=/usr/share/fsl/5.0
+ENV LD_LIBRARY_PATH=/usr/lib/fsl/5.0
```

Comment:
for the moment i decided to resort to this, till we figure out a better thing that not only covers FSL from neurodebian, but also conda and pip, etc.,. i.e niceman. the only thing that can affect this right now, is if someone wanted to use FSL 4.